### DISCUSSION TITLE
Local Media playback through Service call

### DISCUSSION THREAD

**Discussion by jaswanthikolla** [cdnninja/yoto_ha#82] [General]
Is it possible to trigger media playback of local music files?

**Comment by cdnninja** [comment]
Hmmm good question.  Let me mess with the API and find out. Card play api that I have working actually takes a url to the card.  Maybe it would accept other urls too.

**Reply by jaswanthikolla** [reply]
Yoto allows playing of music stream ( radio) that's internet facing, I think they do that only for validation ( validation that it's not HTML page, and it's a stream). I don't think they do any pass through, and connect directly from the device to the music stream.


What we can do is that, we can host the playlist on plex or just stream spotify playlist.

**Reply by cdnninja** [reply]
I tried play card api without success for an mp3. 

I would think would need to find the api for playing radio or uploading mp3.   First do that followed by playing the playlist that is created.
